gm4_heart_canisters/data/gm4/advancements/heart_canisters_max.json

```diff
@@ -0,0 +1,34 @@
+{
+  "display":{
+    "icon":{
+      "item":"player_head",
+      "nbt":"{SkullOwner:{Id:\"25b1ff93-7925-1cd7-53fe-8e6050792daa\",Properties:{textures:[{Value:\"eyJ0aW1lc3RhbXAiOjE0MjkzNDc5OTc3ODcsInByb2ZpbGVJZCI6IjZjZjU0M2E2MGVlOTQzN2NiNjE0YzdiOTRkZTVjNWI3IiwicHJvZmlsZU5hbWUiOiJNcnNNYWtpc3RlaW4iLCJpc1B1YmxpYyI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzQ1NWQ5YmY4NWFjNTY1YjM1ZTFmYTE5MjQ3Y2E2NTQxZGMyZTMzNGJlYzExNWNiNDQ5ZWZiZThlOWI4MTAyMiJ9fX0=\"}]}},display:{Name:\"heart_canisters logo 3\"}}"
+    },
+    "frame": "goal",
+    "title":"All My Loving",
+    "description":{
+      "text":"Max out your Heart Canister health.",
+      "color":"gray"
+    }
+  },
+  "parent":"gm4:heart_canisters_tier_2",
+  "criteria":{
+    "have_all_heart_cans":{
+      "trigger":"minecraft:inventory_changed",
+      "conditions": {
+        "items": [
+          {
+            "item": "minecraft:player_head",
+            "count": 5,
```

Comment:
Does this trigger if the player has 5 total in their inventory, or 5 in a given stack. If it is the former, it might be slightly confusing for a player who gets the achievement but not the health boost.